packages/client/src/components/Question/Questions.tsx

```diff
@@ -264,29 +275,29 @@ const Questions: FunctionComponent<QuestionsProps> = ({
   // Check which questions are causing the need for a permit
   // @TODO: Move this to `imtr-client`
   // See: https://trello.com/c/ZWvyG3Xi/209-refactor-questions-tests-wip
-  let permitsPerQuestion: ClientOutcomes[] = [];
-  checker.permits.forEach((permit: Permit) => {
+  let permitsPerQuestion: imtr.ClientOutcomes[] = [];
+  checker.permits.forEach((permit: imtr.Permit) => {
     const outcomeDecision = permit.getDecisionById("dummy");
 
     if (outcomeDecision) {
       const imtrOutcome = outcomeDecision.getOutput();
-      let outcomeType = ClientOutcomes.PERMIT_FREE;
-
-      if (imtrOutcome === imtrOutcomes.NEED_CONTACT) {
-        outcomeType = ClientOutcomes.NEED_CONTACT;
-      } else if (imtrOutcome === imtrOutcomes.NEED_PERMIT) {
-        outcomeType = ClientOutcomes.NEED_PERMIT;
-      } else if (imtrOutcome === imtrOutcomes.NEED_REPORT) {
-        outcomeType = ClientOutcomes.NEED_REPORT;
+      let outcomeType = imtr.ClientOutcomes.PERMIT_FREE;
+
+      if (imtrOutcome === imtr.imtrOutcomes.NEED_CONTACT) {
```

Comment:
Thinking; `imtr.imtrOutcomes` is a weird name.

Reply:
it is, yeah haha

Reply:
I've updated this to outcomes, so `imtr-client` uses just `outcomes` and the client uses `imtr.outcomes`

Reply:
Please mark this issue as resolved if you like this 👋🏽